smartReceiptsLibrary/src/main/java/co/smartreceipts/android/analytics/events/ErrorEvent.java

```
@@ -29,5 +31,11 @@ public ErrorEvent(@NonNull Class<?> klass, @NonNull Throwable throwable) {
 
     public ErrorEvent(@NonNull String name, @NonNull Throwable throwable) {
         super(Category.OnError, new ImmutableName(name), Collections.singletonList(new DataPoint(TRACE_DATAPOINT_NAME, ExceptionUtils.getStackTrace(throwable))));
+        this.throwable = throwable;
```

Toss a Preconditions.notNull(throwable) around throwable to ensure we fail fast. (If you're curious, I'm happy to have a philosophical discuss around crashing vs not, but I tend to prefer it in the Rx world of things)